sql/core/src/main/scala/org/apache/spark/sql/jdbc/JDBCRDD.scala

```diff
@@ -361,8 +361,14 @@ private[sql] class JDBCRDD(
           val pos = i + 1
           conversions(i) match {
             case BooleanConversion    => mutableRow.setBoolean(i, rs.getBoolean(pos))
```

Comment:
OK i see. Can we add a comment at here to explain that `DateUtils.fromJavaDate` does not handle `null` input?